apps/vanilla/src/pages/index.html

```diff
@@ -6,11 +6,136 @@
     <link rel="stylesheet" href="./index.css" />
     <title>Vite App</title>
   </head>
-  <body class="default-layout">
-    <h1 class="title">Main page ⚡</h1>
-    <div class="links">
-      <a href="example/">Go to example page</a>
-    </div>
-    <script type="module" src="./index.ts"></script>
+  <body class="container">
+    <header class="header font-logo">📚 Books app</header>
```

Comment:
`header` should has it's own font if a logo has one, as this is a header block and it contain a image as a logo, so naming the font to `font-header` would be more correct
```suggestion
    <header class="header font-header">📚 Books app</header>
```

Reply:
this one is also use for the footer so its shouldn't be font-header
![image](https://github.com/saritasa-nest/camp-js-2024-dat-ngo/assets/174302261/6e6abc2b-88a3-4fdb-b104-135dd761dc53)